edenia/eden-spend-explorer
# Feat/delegate view 228

### Update Delegate View

### What does this PR do?

- Resolve #228 
- Resolve #219 

### Steps to test

1. Run project 
2. Enter to Delegate and should see this
![imagen](https://user-images.githubusercontent.com/49013556/202255733-bb0c8cb3-80ca-476c-8633-f6626e50f9bd.png)


#### CheckList
